engine/src/main/java/org/terasology/engine/rendering/AABBRenderer.java

```diff
@@ -215,8 +212,10 @@ private void generateDisplayListWire() {
                 6, 7,
                 7, 4,
         });
+
+        Color color = new Color(Color.black);
         for (int i = 0; i < 8; i++) {
-            meshData.color0.addAll(new float[]{0.0f, 0.0f, 0.0f, 1.0f});
+            meshData.color0.put(color);
```

Comment:
Why do we need to create a new instance of `color` here beforehand, and can't use 
```suggestion
            meshData.color0.put(Color.black);
```
here? Also, do we do a "copy-on-write" here? Otherwise, is it correct that we put the same color instance for all 8 indices?